testprojects/src/python/sources/BUILD

```diff
@@ -12,3 +12,36 @@ resources(
   name='text',
   sources=globs('*.txt'),
 )
+
+resources(
+  name='missing-globs',
+  sources=globs('*.a'),
+)
+
+resources(
+  name='missing-rglobs',
+  sources=rglobs('*.a'),
+)
+
+resources(
+  name='missing-zglobs',
+  sources=zglobs('**/*.a'),
```

Comment:
All of the globs types (`z`/`r`/etc) are converted into one underlying format, so accounting for just "recursive", "non-recursive", and "literal" should be sufficient.

Reply:
Sure, but I'd like to leave the testprojects and integration testing as is until we can close #5863. Does that sound reasonable? I would like to make sure the testing captures the error messages and ensures they display something useful to the user, and all of the skipped tests in the new `test_graph_integration.py` need to be updated when that happens.